### PR TITLE
Podman test

### PR DESCRIPTION
**Pull request by mbiarnes**
doing tests for replacing docker by podman:
- removed docker
- installed Podman

The IP Adress in podman container is only visible if the container is created in root mode. So basically ./build sh and ./start.sh shoudl now be run like:
sudo ./build.sh or sudo ./start.sh.
Also the normal `docker run` should be substitued by  `sudo podman run`

@cristianonicolai : I am not sure how I can "translate" docker-compose commands. I saw in google that there is an implementation podman-compose  that you have to install. But this was not tested.

### CODE REVIEW

File: base/etc/start_business-central-wb.sh [hunk 1]
```diff
@@ -2,5 +2,5 @@
 
 # Start Wildfly with the given arguments.
 echo "Running business-central workbench on JBoss Wildfly..."
-exec ./standalone.sh -b $JBOSS_BIND_ADDRESS -c $KIE_SERVER_PROFILE.xml -Dorg.kie.demo=$KIE_DEMO -Dorg.kie.example=$KIE_DEMO
```
Comment (mareknovotny) [inline base/etc/start_business-central-wb.sh]: why this removal? can you link any change or jira which asked for that?

Comment (mbiarnes) [reply base/etc/start_business-central-wb.sh]: $KIE_DEMO is no where defined.
https://github.com/jboss-dockerfiles/jbpm/commit/409016c3b404ce987aea8707d95b2ca6cdb4d748

Comment (mareknovotny) [reply base/etc/start_business-central-wb.sh]: ok, thanks

File: base/README.md [hunk 2]
```diff
@@ -210,12 +210,23 @@ Troubleshoot
...
 
 	
 Notes
 -----
 
+* Due to internal changes Podman is used instead of docker
+  - all docker commands are still valid
+  - if you want to use command in this script and you're using docker plese replace sudo podman or podman by docker
+  - in case you're using Podman when running a container you have to run it in **root mode** (sudo xxx), otherwise the IP address will
+    not be visible
```
Comment (romartin) [inline base/README.md]: unnecessary new-line

Comment (romartin) [reply base/README.md]: I don't think forcing to create an alias for this is a good idea, see comments above

**Comment by mbiarnes** [comment]
ONLY merge this when we have adapted out environment

**Comment by romartin** [comment]
great thanks @mbiarnes , let us know again when to review this :+1:

**Pull request closed without merging**